docker-compose.production.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.2"
+services:
+ db:
+   image: mysql:5.7
+   env_file:
+     - '.env'
+   ports:
+     - "3306:3306"
+   command: ['mysqld', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+   healthcheck:
+     test: ["CMD", "mysqladmin", "ping", "-h", "localhost","-plinks"]
+     timeout: 20s
+     interval: 10s
+     retries: 10
```

Comment:
this looks identical to the service definition in the main docker-compose file. We only need to override changes.